flask/wrappers.py

```diff
@@ -18,14 +18,94 @@
 _missing = object()
 
 
-def _get_data(req, cache):
+def _get_data(req):
     getter = getattr(req, 'get_data', None)
     if getter is not None:
-        return getter(cache=cache)
```

Comment:
`get_json(cache=False)` shouldn't store any data on the response object and consume any underlying buffers IIRC